src/state/routing/useRoutingAPITrade.ts

```diff
@@ -77,8 +78,11 @@ export function useRoutingAPITrade<TTradeType extends TradeType>(
   const isCurrent = currentTradeResult === tradeResult
 
   return useMemo(() => {
-    if (!amountSpecified || isError || !queryArgs) {
-      return { state: TradeState.INVALID, trade: tradeResult?.trade }
+    if (!shouldGetTrade && amountSpecified) {
+      // If we don't want to fetch new trades, but have valid inputs, return the stale trade.
+      return { state: TradeState.STALE, trade: tradeResult?.trade }
```

Comment:
where is this state used? dont see it anywhere else

Reply:
@tinaszheng it wasn't used, but i thought it might be useful to differentiate between INVALID and STALE since this is technically not invalid